### ISSUE TITLE
loss_modules do not handle loss functions with keyword arguments properly

### ISSUE THREAD

**Issue by twMisc** [twMisc/LM-PyTorch-Training#1]

**Comment by twMisc** [comment]
Should be fixed by d055eeb
